app/middleware/csp.js

```diff
@@ -20,7 +20,8 @@ const CSP_SELF = ["'self'"]
 const frameAndChildSourceCardDetails = ["'self'", 'https://secure-test.worldpay.com/',
   'https://centinelapi.cardinalcommerce.com/']
 
-const imgSourceCardDetails = ["'self'", 'https://www.google-analytics.com/']
+const imgSourceCardDetails = ["'self'", 'https://www.google-analytics.com/',
+  'https://www.gstatic.com/instantbuy/icons/']
```

Comment:
I think we can probably just whitelist `https://www.gstatic.com/` — paths were introduced in CSP2 so we might block people using older browsers. I doubt there’s a huge security difference.